src/Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple/KafkaConsumer.cs

```diff
@@ -22,71 +29,91 @@ public static class KafkaConsumer
                 EnableAutoCommit = false
             }.WithAuthentication(options);
 
-            var serializer = JsonSerializer.CreateDefault(options.JsonSerializerSettings);
+            using var consumer = new ConsumerBuilder<Ignore, string>(config)
+                .SetValueDeserializer(Deserializers.Utf8)
+                .Build();
+            return consumer.Position(new TopicPartition(options.Topic, 0));
```

Comment:
shouldn't partition be at least 1 here?

Reply:
This is identical to the existing code, so I wouldn't change it.